### ISSUE TITLE
typo in /scribe#usage

### ISSUE THREAD

**Issue by icecapp** [codedge-llc/scribe#19]
https://github.com/codedge-llc/scribe#usage
```elixir
iex(1)> data = [%{key: "value", another_key: 123},
...(1)> [%{key: "test", another_key: :key}]
```
extra opening square bracket on second line

**Comment by icecapp** [comment]
otherwise nice package and thank you for open sourcing this

**Comment by hpopp** [comment]
Thanks for the heads up! Feel free to open a PR if you want credit.

**Comment by hpopp** [comment]
Patched in #25
